distributed/tests/test_worker.py

```diff
@@ -507,26 +510,21 @@ async def test_io_loop(s):
         assert w.io_loop is s.loop
 
 
-@gen_cluster(client=True, nthreads=[])
-async def test_spill_to_disk(c, s):
+@gen_cluster(
+    client=True, nthreads=[("", 1)], worker_kwargs={"memory_limit": 1200 / 0.6}
```

Comment:
I recommend setting the limit using a string value like "500MB" (or whatever it is supposed to be). This will be parsed and converted to the proper byte value